includes/class-amp-story-post-type.php

```diff
@@ -1190,7 +1027,7 @@ public static function get_fonts() {
 			static function ( $font ) use ( $fonts_url, $subsets ) {
 				$font['slug'] = sanitize_title( $font['name'] );
 
-				if ( isset( $font['gfont'] ) ) {
+				if ( isset( $font['gfont'] ) && ! empty( $font['gfont'] ) ) {
```

Comment:
When using `empty()`, the `isset` check is redundant.

Reply:
No it's not, my new code sets an empty `gfont`

Reply:
What I meant is that this:

```php
if ( isset( $font['gfont'] ) && ! empty( $font['gfont'] ) ) {}
```

is equal to this:

```php
if ( ! empty( $font['gfont'] ) ) {}
```

`empty()` does not generate a warning if the variable does not exist, so it's safe to remove the `isset` check.

Reply:
I can remove the isset check if you want.

Reply:
Resolved https://github.com/ampproject/amp-wp/pull/3110/commits/e725b9fc55bc16fa7e0614f36b28a900a6d07431